src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -17,17 +17,16 @@
 import seedu.address.model.person.Session;
 import seedu.address.model.person.SessionList;
 import seedu.address.model.person.TimeSlot;
+import seedu.address.model.util.NextSessionUtil;
 
 /**
  * Represents the in-memory model of the address book data.
  */
 public class ModelManager implements Model {
     private static final Logger logger = LogsCenter.getLogger(ModelManager.class);
-
     private final AddressBook addressBook;
     private final UserPrefs userPrefs;
     private final FilteredList<Person> filteredPersons;
-
     private ObservableList<TimeSlot> timeSlots;
     private boolean isDayView;
     private boolean isFullView;
```

Comment:
Not sure why you deleted the blank lines here. In the given code variables are usually split like this ("private static final", "private final" and "private")